common/src/IO/FgdParser.cpp

```diff
@@ -297,7 +297,8 @@ EntityDefinitionClassInfo FgdParser::parseClassInfo(
       classInfo.size = parseSize(status);
     } else if (
       kdl::ci::str_is_equal(typeName, "model") || kdl::ci::str_is_equal(typeName, "studio") ||
-      kdl::ci::str_is_equal(typeName, "studioprop")) {
+      kdl::ci::str_is_equal(typeName, "studioprop") || kdl::ci::str_is_equal(typeName, "sprite") ||
```

Comment:
What happens if an entity has both a model and a sprite? I think in Hammer et al., it "iconsprite" overrides "sprite" and "model", but I'm not sure about that, or what happens if there is both a "sprite" and a "model" property. Now in TB, you will get a warning and whatever came last in the file, wins.

In any case, we should specify the correct expected behavior with a test.

Reply:
I'm not sure, as far as I know they are all aliases of the same behaviour and there's no point in specifying multiple in the FGD. In Sledge I prioritised sprite over iconsprite but I never tested it in any other editor.